docs/schema.mdx

```diff
@@ -140,3 +141,9 @@ const credentialsList: Credentials = [
 }
 
 ```
+
+## Schema Library
```

Comment:
I would say probably should be featured higher on the page. I would say just under Introduction would be best. Definitely want this to be front and center and the can scroll down for the more wordy explanation. But at the very least this ought to be above Example I think.

Reply:
Indeed it looks better above example section.